compiler/opcodes.go

```diff
@@ -48,8 +48,11 @@ const (
 	OpSetLocal                       // Set local variable
 	OpDefineLocal                    // Define local variable
 	OpSetSelLocal                    // Set local variable using selectors
+	OpGetFreePtr                     // Get free variable pointer object
 	OpGetFree                        // Get free variables
 	OpSetFree                        // Set free variables
+	OpGetLocalPtr                    // Get local variable as a pointer
+	OpSetLocalPtr                    // Set local variable as a pointer
```

Comment:
I noticed that `OpSetLocalPtr` and `OpGetLocalPtr` can be combined into one opcode as they're always used together. But, when I tested with combined opcode, for some reasons, the benchmark was **slower** (adding 0.2-0.4 seconds) even though fib(35) does not even use them. Combined code looks like this:
```
			localIndex := int(v.curInsts[v.ip+1])
			v.ip++

			if v.curFrame.localFreeVars == nil {
				v.curFrame.localFreeVars = make(map[int]*objects.FreeVar)
			}

			sp := v.curFrame.basePointer + localIndex

			val := v.stack[sp]

			if obj, ok := val.(*objects.FreeVar); ok {
				v.curFrame.localFreeVars[localIndex] = obj
			} else {
				v.curFrame.localFreeVars[localIndex] = &objects.FreeVar{Value: &val}
			}

			v.stack[sp] = v.curFrame.localFreeVars[localIndex]

			if v.sp >= StackSize {
				v.err = ErrStackOverflow
				return
			}

			v.stack[v.sp] = v.curFrame.localFreeVars[localIndex]
			v.sp++
```
Definitely something to investigate further.